modules/dhcp_libvirt/subnet_service_initializer.rb

```diff
@@ -37,7 +37,7 @@ def parse_config_for_subnets
       raise Proxy::DHCP::Error("Only one subnet is supported") if ret_val.size > 1
       ret_val
     rescue Exception => e
-      msg = "Unable to parse subnets XML"
+      msg = "Unable to parse subnets XML: #{e}"
```

Comment:
Doesn't the `, e` to `logger.error` already add this information?

Reply:
Yes but apparently I want the error message to be returned to the client. Our error reporting is sometimes very weak.

Reply:
Would `logger.exception` be useful here (and later in the file)?

Reply:
Is the specific exception message still needed when using `logger.exception`? I think that already formats the message to include the specific exception message.